Group_notes

```diff
@@ -0,0 +1,21 @@
+Group meeting notes
+
+First, testing to see if we can successfully push a text file to our personal diagnostics folder, and then to the folder of our group within Github
+
+What to detect / measure of interest:
+- mean intensity/BOLD value
+
+Outlier detection methods:
+- standard deviation: mean +/- (std. dev. * multiplier)
+- interquartile range: upper = Q1/Q3 +/- (IQR * multiplier)
+- dvars (sum square of variance)
```

Comment:
Good to distinguish between a measure (such as dvars) that you can use to detect outliers on, and an outlier detection method, such as standard deviation or interquartile range.